.github/workflows/create-release.yml

```diff
@@ -1,5 +1,7 @@
 name: Create Release Pull Request
 on:
+  # THIS WORKFLOW SHOULD NEVER BE TRIGGERED ON A PUSH EVENT. IF TRIGGERED ON A
+  # PUSH EVENT IT MAY CREATE AN ENDLESS STREAM OF 'version bump' COMMITS.
```

Comment:
I wonder if we can/should enforce this by adding something like the following somewhere in this workflow :thinking: 

```yaml
if: ${{ github.event != "push" }}
```

Reply:
That comment is very difficult to ignore, so if someone adds the push event to it, they could just delete that `if`.

Reply:
True, my thinking was that it's easier to notice the `if` being removed in a review than it is to notice the `on: push` being added regardless of the comment.

From the perspective of the person making the change, this may be more useful. From the perspective of a reviewer, the `if` may be more helpful. We could have both.

Reply:
I'm fine with adding it then